admin_manual/configuration/server/security/ransomeware-protection.rst

```diff
@@ -0,0 +1,95 @@
+                                                                    anomalies in a user account. It enables an administrator to determine the point in time
+                                                                    where undesired actions happened as a
+                                                                    prerequisite for restoration.
+Ransomguard Restorer ``occ ransomguard:restore <timestamp> <user>`` A command for administrators to revert all
+                                                                    operations in a user account that occurred after
+                                                                    a certain point in time.
+==================== ============================================== ================================================
+
+.. note:: 
+   ``<timestamp>`` must be in `the Linux timestamp format`.
+
+Requirements
+~~~~~~~~~~~~
+
+#. **Ransomware Protection.** It records the move operations so that it can revert them in the case of an attack. Other operations are revertible using ownCloud's existing database tables.
+#. **ownCloud Versions App.** It is required to restore older file versions and because capabilities of Ransomware Protection depend on its configuration.
+#. **ownCloud Trash Bin App.** It is required both to restore deleted files and because capabilities of Ransomware Protection depend on its configuration.
+
+Limitations
+~~~~~~~~~~~
+
+-  Ransomware Protection works with master-key based storage encryption. Credential-based storage encryption only Ransomware Prevention (Blocking) works.
+-  Rollback is not based on snapshots.
+-  The `trash bin retention policy`_ may delete files, making them unrecoverable. To avoid this, set ``trashbin\_retention\_obligation`` to ``disabled``, or choose a conservative policy for trash bin retention. However, please be aware that this may increase storage requirements.
+-  Trash bin items may be deleted by the user making them unrecoverable by Ransomware Protection => Users need to know this.
+-  Versions have `a built-in "thin-out" policy`_ which makes it possible that required file versions are unrecoverable by Ransomware Protection. To help avoid this, set ``versions\_retention\_obligation`` to ``disabled`` or choose a conservative policy for version retention. Please be aware that this might increase your storage needs.
+-  A specific version of a file that is needed for rollback might have been manually restored, making this version potentially unrecoverable by Ransomware Protection. Currently, after restoration the restored version `is not a version anymore`, e.g., the version is not present in versioning.
+-  Contents in secondary storages, such as *Windows network drives*, *Dropbox*, and *Google Drive*, are unrecoverable by Ransomware Protection, because they do not have versioning or trash bin enabled in ownCloud. For example, for WND we might have versions, but for others such as Google Drive, Dropbox we don't.
```

Comment:
Please remove the second part of this point. We just say that it will only work for files stored on ownCloud's primary storage.